modules/persistent-storage-csi-migration-enable.adoc

```diff
@@ -31,7 +31,7 @@ spec:
   featureSet: TechPreviewNoUpgrade <1>
 ...
 ----
-<1> Enables automatic migration for AWS EBS, Cinder, and Azure Disk.
+<1> Enables automatic migration for AWS EBS, Cinder, Azure Disk, and Azure File.
```

Comment:
With this line, we might get an incorrect sentence after https://github.com/openshift/openshift-docs/pull/38104 merges.
The `, and Azure File` will become `, and Azure File, and VMware vSphere.`.

We need to merge them in a synchronous way (first merge this PR, then update the other one, then merge it).